openquake/hazardlib/imt.py

```diff
@@ -348,13 +348,20 @@ def LiqOccur():
     return IMT('LiqOccur')
 
 
-def LSE():
+def LqSE():
```

Comment:
Same comment as above regarding `LiqProb()` and `LiqOccur()` →`LqProb()` and `LqOccur()`

Also, @micheles probably no one noticed this until now, but if you check above, the return statement for the following IMT seems wrong..

```py
def DispProb():
    """
    Displacement probability
    """
    return IMT('RSD595')
```

Reply:
Yes, I would have expected a `return IMT('DispProb')`